lms/templates/help_modal.html

```diff
@@ -18,7 +18,7 @@
   <a href="#help-modal" rel="leanModal" role="button">${_("Support")}</a>
 </div>
 
-<section id="help-modal" class="modal" aria-hidden="true" role="dialog" tabindex="-1" aria-label="${_("{platform_name} Support").format(platform_name=static.get_platform_name())}">
+<section id="help-modal" class="modal" aria-hidden="true" role="dialog" aria-modal="true" tabindex="-1" aria-label="${_("{platform_name} Support").format(platform_name=static.get_platform_name())}">
```

Comment:
@davec-edx can we make this a ```<div>``` instead of a ```<section>``` ?  Technically, this is not a document section (re would be included in a TOC).   Also, for simplification, you may want to convert the ```aria-label``` to an ```aria-labelledby``` attribute which takes an IDref instead of a string.  Since the label already exists down below on line 34, you can put an ID on that ```<h2>``` element and reference it for the dialog label.